dependencies.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from database import SessionLocal
+
+
+async def get_db() -> AsyncSession:
+    db = SessionLocal()
```

Comment:
Replace `SessionLocal()` with `async_session()` to correctly create a new session.